script-test/playbackstrategies/msestrategytest.js

```diff
@@ -48,16 +48,21 @@ require(
           windowEndTime: windowEndTimeMS || 0
         };
 
-        mockVideoElement = document.createElement('video');
-        mockVideoElement.currentTime = 0;
-        spyOn(mockVideoElement, 'addEventListener');
-        spyOn(mockVideoElement, 'removeEventListener');
+        if (defaultMediaKind === MediaKinds.AUDIO) {
+          mockMediaElement = document.createElement('audio');
```

Comment:
Might be better to always create both a video and audio mock element, so when you assert on it you're sure it's the correct one. Then it's up to the mocking of createElement to use the correct one.